make/tests

```diff
@@ -40,6 +40,14 @@ test/unit/libmultiple.so : test/unit/multiple_translation_units1.o test/unit/mul
 
 test/unit/multiple_translation_units_test.cpp : test/unit/libmultiple.so
 
+
+DEBUG_TESTS := $(subst .cpp,$(EXE),$(shell find test -name *debug_*test.cpp))
```

Comment:
What is this? It wasn't included in the description of the pull request. It seems redundant?

Reply:
In case in the future we want do something special of some diagnostic/debug tests. Single them out and change/add flags.

Reply:
Let's remove this if it's for unspecified future use.

Reply:
Done